README.md

```diff
@@ -46,13 +47,20 @@ As a Maker
 So that I can avoid others posting messages on Chitter as me
 I want to log out of Chitter
 
+not completed:
 ADVANCED
 
 As a Maker
 So that I can stay constantly tapped in to the shouty box of Chitter
 I want to receive an email if I am tagged in a Peep
 ```
-
+```
```

Comment:
maybe in your readme, you could have also said what the first url '/' route the user should go to first to get the web side of things working.